tests/python/ci/test_ci.py

```diff
@@ -703,27 +713,31 @@ def run(type, data, check):
     )
 
     run(
-        type="ISSUE",
+        source_type="ISSUE",
         data={
             "title": "[something] A title",
             "number": 1234,
             "user": {
                 "login": "person5",
             },
             "labels": [{"name": "something2"}],
+            # pylint: disable=line-too-long
             "body": textwrap.dedent(
                 """
                 `mold` and `lld` can be a much faster alternative to `ld` from gcc. We should modify our CMakeLists.txt to detect and use these when possible. cc @person1
```

Comment:
Can we not have a line per sentence here?

Reply:
we could but I would argue that makes this harder to read and isn't the idiomatic way to include a long message, hence the linter disable

Reply:
i think we should just be consistent across whatever we do on line 734 as well

Reply:
Agree with @areusch we should be consistent. The reason I'd err on the side of splitting is so it's easier to read on as many peoples screens as possible, without side scrolling or the editor trying to line wrap it.